Foundation/NSStringAPI.swift

```diff
@@ -1894,6 +1894,14 @@ extension StringProtocol {
 
 }
 
+// Old method names
+extension NSString {
+    @available(*, unavailable, message: "Use path(withComponents:) instead.")
```

Comment:
Should this use `renamed: path(withComponents:)` to get a fix-it instead of `message: ...`?

Reply:
Updated!